static/style.css

```diff
@@ -3,112 +3,350 @@
+  align-items: center;
+  justify-content: center;
 }
 
-.head div {
-  margin-left: 20px;
-  margin-right: 20px;
+.languages ul li {
+  margin: 13px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
-.logo img {
-  width: 60px;
-  border-radius: 50%;
+.introduction ul li {
+  margin-right: 30px;
 }
 
-.name h1 {
-  font-size: 56px;
-  font-family: 'Crete Round', Arial, Helvetica, sans-serif;
-  color: #ff6b00;
+.links ul li {
+  margin-right: 12px;
+  padding: 10px;
+  background-color: #fff;
+  font-size: 15px;
+  font-weight: 500px;
+  line-height: 20px;
+  color: #091e42;
+  font-family: 'Inter', sans-serif;
+  border: #8993a4 solid 1px;
 }
 
-.name h3 {
-  font-size: 24px;
-  font-family: Arial, Helvetica, sans-serif;
+.languages ul li ul li {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  display: flex;
+  align-items: center;
+  text-align: center;
   color: #fff;
 }
 
-.content {
-  width: 80vw;
+.languages ul li ul {
+  margin-top: 24px;
+}
+
+.introduction .social ul {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+}
+
+.margin_bottom {
+  margin-bottom: 20px;
+}
+
+.padding_lr {
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+/* Recent works */
+.recent_works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   padding-top: 120px;
-  padding-left: 30px;
-  padding-right: 50px;
 }
 
-.description {
-  margin-top: 10px;
+.recent_works .title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 327px;
+  height: 83px;
+  padding-bottom: 24px;
+  border-bottom: #212121 solid 1px;
+}
+
+.recent_works .title h1 {
+  font-size: 40px;
+  font-weight: 400;
+  color: #1c1a19;
+  font-family: 'Crete Round', serif;
+  padding-bottom: 24px;
 }
 
-.description p {
+.recent_works .place_holder {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  padding-top: 80px;
+}
+
+.recent_works .place_holder img {
+  width: 100%;
+}
+
+.mp_stories {
+  display: flex;
+  flex-direction: column;
+  margin-top: 24px;
+}
+
+.mp_stories h2 {
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  color: #091e42;
+  font-family: 'Crete Round', serif;
+}
+
+.mp_stories p {
   font-size: 16px;
-  font-family: 'Popins', Arial, Helvetica, sans-serif;
-  letter-spacing: 0.7px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #091e42;
+  font-family: "Inter", sans-serif;
+}
+
+.links {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding-top: 12px;
 }
 
-.menu img:hover {
-  background-color: gray;
+.buttons {
+  width: 100%;
+  margin-top: 12px;
+  padding-top: 12px;
 }
 
-.menu {
-  margin-right: 20px;
+.buttons a {
+  padding: 12px;
+  color: #fff;
+  background-color: #ff6b00;
+  text-decoration: none;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
 }
```

Comment:
- [ ] Kindly implement the `hover`, `active`, `pressed` and `disabled` pseudo-classes for all the buttons in your webpage including the **_Get my resume_** button. The screenshot below represents the section of the page on the Figma template that you chose where you can find the style declarations for the various pseudo-classes that should be implemented. When you click on each category, you will get the style declarations listed in the _**Inspect**_ pane to the right on Figma 😄 💻:

![image](https://github.com/wacoo/Portfolio-Project/assets/77106962/fc42b00a-a99c-48d3-a3dd-599c47a9e41f)